src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -574,6 +574,31 @@ internal bool IsNew
             get { return (_modifiers & DeclarationModifiers.New) != 0; }
         }
 
+        internal sealed override bool IsReplace
+        {
+            get { return (_modifiers & DeclarationModifiers.Replace) != 0; }
+        }
```

Comment:
Replaced properties and events are not emitted to metadata, only the accessors are.